### PR TITLE
Rdb stall

### PR DESCRIPTION
**Pull request by andrewMorrison95**
1. Change rdb cluster type to RDB to allow use of new rdb cluster on finspace.
2. Change replaylog to 0b to disable intia;l replay when process comes up (as this will be prior to period start)
3. Change upd to discard any incoming data (this will be reverted later at period start/end)
4. Add wdb and rdb to the conneciton list for the new rdb in order to facilitate sending 'ready' message.

### CODE REVIEW

File: appconfig/settings/wdb.q [hunk 1]
```diff
@@ -2,4 +2,3 @@
 \d .wdb
 replay:0b;     // We don't want to replay as the wdb will start before the period it is meant to subscribe to
 upd:{[t;x]};   //Discard data until the end of period signals the start of new period
-
```
Comment (jonathonmcmurray) [inline appconfig/settings/wdb.q:5]: remove unrelated whitespace change please

Comment (andrewMorrison95) [reply appconfig/settings/wdb.q:5]: should be fixed now

File: appconfig/settings/rdb.q [hunk 2]
```diff
@@ -14,10 +14,15 @@ timeout:system"T"
...
+
 
 hdbtypes:()                         //connection to HDB not needed
 
 subfiltered:0b
 // path to rdbsub{i}.csv
 subcsv:hsym first `.proc.getconfigfile["rdbsub/rdbsub",(3_string .proc`procname),".csv"];
+
+\d .servers
+CONNECTIONS:`rdb`wdb`gateway         // if connectonstart false,include tickerplant in tickerplanttypes, not in CONNECTIONS
```
Comment (zanbuchanan) [inline appconfig/settings/rdb.q:28]: Do we need the gateway?

Comment (andrewMorrison95) [reply appconfig/settings/rdb.q]: removed

**Review by jonathonmcmurray — APPROVED** [review]
